proposals/graduation/falco.md

```diff
@@ -0,0 +1,78 @@
+# Falco Graduation Proposal
+
+Falco, the Runtime Security Project, joined the CNCF in October 2018 as a Sandbox Project.
+
+On January 8th, 2020 Falco was [promoted to the Incubating stage](https://www.cncf.io/blog/2020/01/08/toc-votes-to-move-falco-into-cncf-incubator/). Since then, the Falco community, users and maintainers continued to actively invest in Falco, fueling sustained growth in terms of adopters, governance, contributions, features and integrations with other CNCF projects.
+
+At this stage, we believe that Falco is ready for the next natural step: Graduation.
+
+Key highlights as of April 19th, 2021:
+
+- Github stars: 4.200+
+- Core repository contributors: 100+
+- Forks: 660+
+- Releases: 110
+- Slack channel members: 750+
+- Downloads:
+  - 24M+ Docker hub pulls (+570% since Incubation)
+  - 30M+ driver downloads from our CDN (+680% since Incubation)
+	- Since Falco compiles a driver for the host when it starts (and fallbacks to download a prebuilt one) this is a very conservative proxy of how many times Falco is installed
+- Top 3 media mentioned Incubated CNCF project for last 3 months (January, February, March 2021)
+- Keynote mention at Kubecon+CloudNativeCon 2020 -- Constance Caramolis ([youtube](https://www.youtube.com/watch?v=uNpZA5fz5o8))
+- End user talk at Kubecon 2020 -- Shane Lawrence, Shopify ([youtube](https://www.youtube.com/watch?v=rBqBrYESryY))
+- End user talk at Kubecon 2020 -- Natch Ruengsakulrach & Eric Hollis, MathWorks ([slides](https://static.sched.com/hosted_files/kccncna20/aa/KubeCon_NA_Virtual_2020-Cyber_Kill_Chain_Falco.pdf))
+- Inclusion in CNCF CKS certification
+- Several vendors have built commercial offerings on top of Falco (e.g. [GitLab](https://about.gitlab.com/blog/2020/08/18/how-gitlab-can-help-you-secure-your-cloud-native-applications/), [SumoLogic](https://www.sumologic.com/solutions/kubernetes/))
```

Comment:
- To enable wider adoption of the product the Falco website content is translated in Chinese, Korean, Japanese, and Malayalam languages.

Reply:
A huge +1 from me!!!